src/test/java/org/folio/rest/api/LoansApiTest.java

```diff
@@ -107,9 +109,10 @@ public void canCreateALoan()
       .withOverdueFinePolicyId(overdueFinePolicyId)
       .withLostItemPolicyId(lostItemPolicyId)
       .withClaimedReturnedDate(claimedReturnedDate)
+      .withAgedToLostDate(agedToLostDate)
       .withAgedToLostDelayedBilling(new JsonObject()
-            .put("lostItemHasBeenBilled", Boolean.FALSE)
-            .put("dateLostItemShouldBeBilled", dateLostItemShouldBeBilled.toString()))
+        .put("lostItemHasBeenBilled", Boolean.FALSE)
```

Comment:
Given this line is being changed, personally I would statically import the Boolean constant

```suggestion
        .put("lostItemHasBeenBilled", FALSE)
```

Reply:
I've changed it to primitive boolean.